src/controller/CHIPDeviceController.h

```diff
@@ -69,7 +70,7 @@ namespace Controller {
 
 using namespace chip::Protocols::UserDirectedCommissioning;
 
-constexpr uint16_t kNumMaxActiveDevices = 64;
+constexpr uint16_t kNumMaxActiveDevices = CHIP_CONFIG_CONTROLLER_MAX_ACTIVE_DEVICES;
 constexpr uint16_t kNumMaxPairedDevices = 128;
```

Comment:
Shall we also make kNumMaxPairedDevices configurable?

Reply:
Personally, I think we shouldn't have a limit for the number of paired devices. While it's quite reasonable to limit the number of active devices, the controller class should be able to control devices even if they were paired by a different commissioner, in which case maintaining the list of "commissioned" devices wouldn't make much sense to me.